tests/unit/test-cmd/test.build.js

```diff
@@ -320,6 +322,44 @@ describe('build', () => {
     }
   ));
 
+  it('raises a UsageError if zip file exists', () => {
+    return withTempDir(
+      (tmpDir) => {
+        const testFileName =
+          path.join(tmpDir.path(), 'minimal_extension-1.0.zip');
+        return fs.writeFile(testFileName, 'test')
+          .then(() =>
+            build({
```

Comment:
can you move `build({` to the previous line? it makes much more clear that the arrow function is going to return the result of the `build` call.